blog/README.md

```diff
@@ -0,0 +1,232 @@
+[1/2] STEP 2/6: ARG KERNEL_VERSION
+--> Using cache 0a1ec1eb4b3eb2accba3475b2e347ac81c0f01b607d25c5e81c0d01471092d21
+--> 0a1ec1eb4b3
+[1/2] STEP 3/6: WORKDIR /build/
+--> Using cache 02fd2db894dbf7f2c59b7dbabf64c2114eb283ddd230163fb9bd8df892045e02
+--> 02fd2db894d
+[1/2] STEP 4/6: RUN git clone https://github.com/openshift-psap/simple-kmod.git
+--> Using cache 419fda4f52482891200333bfa165ab40d66c40bd4180be1c207dd48ba6fa7d27
+--> 419fda4f524
+[1/2] STEP 5/6: WORKDIR /build/simple-kmod
+--> Using cache 80d50b672deb18ba0f3876e32d487eaaa5832765c87f724c079130576c8cfd6d
+--> 80d50b672de
+[1/2] STEP 6/6: RUN make all install KVER=4.18.0-372.53.1.el8_6.x86_64
+make -C /lib/modules/4.18.0-372.53.1.el8_6.x86_64/build M=/build/simple-kmod EXTRA_CFLAGS=-DKMODVER=\\\"e852852\\\" modules
+make[1]: Entering directory '/usr/src/kernels/4.18.0-372.53.1.el8_6.x86_64'
+  CC [M]  /build/simple-kmod/simple-kmod.o
+  CC [M]  /build/simple-kmod/simple-procfs-kmod.o
+  Building modules, stage 2.
+  MODPOST 2 modules
+  CC      /build/simple-kmod/simple-kmod.mod.o
+  LD [M]  /build/simple-kmod/simple-kmod.ko
+  CC      /build/simple-kmod/simple-procfs-kmod.mod.o
+  LD [M]  /build/simple-kmod/simple-procfs-kmod.ko
+make[1]: Leaving directory '/usr/src/kernels/4.18.0-372.53.1.el8_6.x86_64'
+gcc -o spkut ./simple-procfs-kmod-userspace-tool.c
+install -v -m 755 spkut /bin/
+'spkut' -> '/bin/spkut'
+install -v -m 755 -d /lib/modules/4.18.0-372.53.1.el8_6.x86_64/
+install -v -m 644 simple-kmod.ko        /lib/modules/4.18.0-372.53.1.el8_6.x86_64/simple-kmod.ko
+'simple-kmod.ko' -> '/lib/modules/4.18.0-372.53.1.el8_6.x86_64/simple-kmod.ko'
+install -v -m 644 simple-procfs-kmod.ko /lib/modules/4.18.0-372.53.1.el8_6.x86_64/simple-procfs-kmod.ko
+'simple-procfs-kmod.ko' -> '/lib/modules/4.18.0-372.53.1.el8_6.x86_64/simple-procfs-kmod.ko'
+depmod -F /lib/modules/4.18.0-372.53.1.el8_6.x86_64/System.map 4.18.0-372.53.1.el8_6.x86_64
+--> 26dfedc51be
+[2/2] STEP 1/5: FROM quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:127670aaa6073e97bd6d919e1b57bd16b60435410b096aeefce7cd64f6c83d24
+[2/2] STEP 2/5: ARG KERNEL_VERSION
+--> 2879805a870
+[2/2] STEP 3/5: COPY --from=builder /etc/driver-toolkit-release.json /etc/
+--> 8dea5bb2bd8
+[2/2] STEP 4/5: COPY --from=builder /lib/modules/4.18.0-372.53.1.el8_6.x86_64/*.ko /usr/lib/modules/4.18.0-372.53.1.el8_6.x86_64/
+--> 6249995c281
+[2/2] STEP 5/5: RUN depmod -a 4.18.0-372.53.1.el8_6.x86_64 && echo simple_kmod > /etc/modules-load.d/simple_kmod.conf
+[2/2] COMMIT quay.io/ybettan/coreos-layering:simple-kmod
+--> 612c8c8a54f
+Successfully tagged quay.io/ybettan/coreos-layering:simple-kmod
+612c8c8a54f0800cf8dcb4c20d5d3d384dc47463711d6d3ad1bd93d52e982b1e
+
+```
+
+Now, let's push the image to some image registry in order for the image to be accesible
+by the cluster.
+```
+$ podman push quay.io/ybettan/coreos-layering:simple-kmod
+```
+
+Make sure the image is public or it have the correct pull-secret to pull the image.
```

Comment:
Maybe you could also link the respective OCP [doc](https://docs.openshift.com/container-platform/4.12/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets) regarding the global pull-secret. This way users will have all the info they need to make this work.

Reply:
Good idea. I will add it indeed.

Reply:
Added.